dirs/dirs.go

```diff
@@ -145,12 +145,16 @@ const (
 	CoreLibExecDir   = "/usr/lib/snapd"
 	CoreSnapMountDir = "/snap"
 
-	// Directory with snap data inside user's home
+	// UserHomeSnapDir is the directory with snap data inside user's home
 	UserHomeSnapDir = "snap"
 
 	// HiddenSnapDataHomeDir is an experimental hidden directory for snap data
 	HiddenSnapDataHomeDir = ".snap/data"
 
+	// ExposedSnapDir is the directory where snaps should place user-facing
+	// data after ~/snap has been migrated to ~/.snap
```

Comment:
nitpick: I guess it's obvious, but it might be better to specify that this is relative to the user's `$HOME`.